src/shared/database/setup.ts

```diff
@@ -185,4 +185,10 @@ async function setupNetworksTable(): Promise<void> {
         .catch((err: Error) => log.error(err.message))
     }
   })
+  if (networksIds.includes('nft-dev')) {
```

Comment:
Will this cause foreign key errors?

Reply:
It doesnt use a foreign key. It just check whether nft-dev is in the array of ids of current networks on the table and if so delete that row using knex.

Reply:
I don't think the networks table is used as a foreign key anywhere